tests/PSinLinuxCloudShellImage.Tests.ps1

```diff
@@ -1,4 +1,15 @@
+
+
+
 Describe "Various programs installed with expected versions" {
+ 
+    It "experiment" {
```

Comment:
Do we still need this test?

Reply:
well spotted, will remove in a future update